farmerbot/internal/farmerbot.go

```diff
@@ -328,11 +333,39 @@ func (f *FarmerBot) iterateOnNodes(ctx context.Context, subConn Substrate) error
 	return nil
 }
 
+func addPriorityToNodes(priorityNodes, farmNodes []uint32) []uint32 {
+	updatedFarmNodes := make([]uint32, len(farmNodes))
+	copy(updatedFarmNodes, farmNodes)
+
+	updatedPriorityNodes := make([]uint32, len(priorityNodes))
+	copy(updatedPriorityNodes, priorityNodes)
```

Comment:
I think the copy and delete part should be replaced with empty slices and append nodes if exist.